opentelekomcloud/services/nat/resource_opentelekomcloud_nat_gateway_v2.go

```diff
@@ -118,7 +118,7 @@ func resourceNatGatewayV2Create(ctx context.Context, d *schema.ResourceData, met
 
 	// set tags
 	tagRaw := d.Get("tags").(map[string]interface{})
```

Comment:
would be better
```go
tagRaw := d.Get("tags").(map[string]interface{})
if len(tagRaw) > 0 {
    log.Printf("[WARN] NAT gatewat in `eu-ch2` doesn't support tagging)
}
if len(tagRaw) > 0 && config.GetRegion(d) != "eu-ch2" {
```

Reply:
in my opinions it will spam the users who never heard about eu-ch2